spark/src/test/scala/ai/chronon/spark/test/JoinUtilsTest.scala

```diff
@@ -30,13 +30,13 @@ import org.junit.Assert._
 import org.junit.Test
 
 import scala.collection.mutable
-import scala.util.Try
+import scala.util.{Random, Try}
 
 class JoinUtilsTest {
 
   lazy val spark: SparkSession = SparkSessionBuilder.build("JoinUtilsTest", local = true)
   private val tableUtils = TableUtils(spark)
-  private val namespace = "joinUtil"
+  private val namespace = "joinUtil"  + "_" + Random.alphanumeric.take(6).mkString
```

Comment:
similar to above comment
hard-coded references might need to get the random string as well. 

https://sourcegraph.d.musta.ch/github.com/airbnb/chronon/-/blob/spark/src/test/scala/ai/chronon/spark/test/JoinUtilsTest.scala?L239-240

https://sourcegraph.d.musta.ch/github.com/airbnb/chronon/-/blob/spark/src/test/scala/ai/chronon/spark/test/JoinUtilsTest.scala?L277-279

https://sourcegraph.d.musta.ch/github.com/airbnb/chronon/-/blob/spark/src/test/scala/ai/chronon/spark/test/JoinUtilsTest.scala?L334

https://sourcegraph.d.musta.ch/github.com/airbnb/chronon/-/blob/spark/src/test/scala/ai/chronon/spark/test/JoinUtilsTest.scala?L351